src/lib.rs

```diff
@@ -867,6 +867,9 @@ impl Socket {
         (_, set_identity) => ZMQ_IDENTITY as &[u8],
         (_, set_subscribe) => ZMQ_SUBSCRIBE as &[u8],
         (_, set_unsubscribe) => ZMQ_UNSUBSCRIBE as &[u8],
+        (_, set_xpub_verbose) => ZMQ_XPUB_VERBOSE as i32,
```

Comment:
The `ZMQ_XPUB_VERBOSE`, `ZMQ_XPUB_MANUAL` and `ZMQ_INVERT_MATCHING` options all take a (semantically) boolean value, not an `i32`. The sockopt machinery will convert `true` and `false` on the Rust side to `1` and `0` on the C side automatically.